ansible/roles/infra-osp-template-generate/tasks/locate_template.yml

```diff
@@ -21,16 +21,15 @@
     cloud_infra_nested_template_src: "../configs/{{ env_type }}/files/cloud_providers/{{cloud_provider}}_cloud_template_nested.j2"
   when: stat_nested_template.stat.exists
 
-# Until there is a default template, leave this disabled and just fail instead
-# - name: Use the default infra template
-#   set_fact:
-#     cloud_infra_template_src: "templates/cloud_template.j2"
-#   when: not stat_local_template.stat.exists
+- name: Use the default infra template
+  set_fact:
+    cloud_infra_master_template_src: "templates/cloud_template_master.j2"
```

Comment:
I'm not seeing this folder or file in the role directory?

Reply:
Because i forgot to push. Done.